docs/resources/gluerecord.md

```diff
@@ -0,0 +1,23 @@
+# Resource: gluerecord
```

Comment:
After authoring this, I've realized that a data record would be useful, but I think I'd need to add a get request in the glue record in domain.go to request specific glue records, rather than [list all.](https://github.com/go-gandi/go-gandi/blob/master/domain/domain.go#L91-L94)

Reply:
Yes, you need to add a Get request.

Reply:
Note the data part could also come in anther PR i guess.

Reply:
Updated to integrate the Get request as opposed to the list all.  Added the data part as well for completeness.